src/main/java/nextstep/subway/application/LineService.java

```diff
@@ -67,13 +68,21 @@ public void deleteLine(final Long id) {
     @Transactional
     public void addSections(final Long id, final SectionRequest sectionRequest) {
         final Line line = findLineById(id);
-        final Section section = new Section(findStationById(sectionRequest.getUpStationId()),
-                findStationById(sectionRequest.getDownStationId()), sectionRequest.getDistance());
+        final Station upStation = findStationById(sectionRequest.getUpStationId());
+        final Station downStation = findStationById(sectionRequest.getDownStationId());
+        final Section section = sectionRequest.toSection(line, upStation, downStation);
         line.addSection(section);
     }
 
     private Station findStationById(final long id) {
         return stationRepository.findById(id)
                 .orElseThrow(() -> new NoSuchElementException("지하철역을 찾을 수 없습니다."));
     }
+
+    @Transactional
+    public void deleteStation(final Long id, final Long stationId) {
```

Comment:
전반적으로 매개변수로 wrapper 클래스를 사용하고 있는데요.
`nul`l이 들어오지 않는 것이 예상된다면 primitive 타입을 사용하는게 안전해보입니다.
```suggestion
    public void deleteStation(final long id, final long stationId) {
```

Reply:
id wrapper 클래스를 primitive 타입으로 변경 하겠습니다.